_data/comunidades/comunidad-madrid.json

```
@@ -47,10 +47,15 @@
       "name": "Boletín oficial Comunidad Madrid",
       "twitter": "ComunidadMadrid"
     },
+    {
+      "url": "cjcm.es",
+      "name": "Consejo de la Juventud de la Comunidad de Madrid",
+      "twitter": "ComunidadMadrid"
```

Creo que este es el correcto

```suggestion
      "twitter": "_CJCM_"
```